upload-artifact/action.yml

```
@@ -0,0 +1,68 @@
+name: Upload 
+description: Upload artifacts to either our own servers or GitHub
+
+inputs:
+  destination:
+    description: Where to store the artifact (github or jahia)
+    required: true
```

Maybe we could use an enum to limit the possibilities and errors (https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs)?
```suggestion
  destination:
    description: Where to store the artifact (github or jahia)
    required: true
    type: choice
    options:
    - github
    - jahia
```